homework/01LessonOne.js

```diff
@@ -107,6 +108,10 @@ var notNotTrue;
 //
 // Your Answer Goes Here:
 //
+// The && (and) operator results in a true value only if both variables in the argument are true.
+// The || (or) operator results in a true value if only one of the variables in the argument are true.
+
+// Logical operators are typically used with Boolean (logical) values. When they are, they return a Boolean value. However, the && and || operators actually return the value of one of the specified operands, so if these operators are used with non-Boolean values, they may return a non-Boolean value.
 // ****
```

Comment:
Excellent for you to point this out. `&&` and `||` can be used in really interesting ways once you understand how they work under the hood.